library/entity/points_of_interest/src/main/java/org/quiltmc/qsl/points_of_interest/api/PointOfInterestHelper.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2016, 2017, 2018, 2019 FabricMC
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.points_of_interest.api;
+
+import static net.minecraft.world.poi.PointOfInterestTypes.STATE_TO_TYPE;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.util.Holder;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.Util;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.util.registry.RegistryKey;
+import net.minecraft.world.poi.PointOfInterest;
+import net.minecraft.world.poi.PointOfInterestType;
+import net.minecraft.world.poi.PointOfInterestTypes;
+
+/**
+ * This class provides utilities to create a {@link PointOfInterestType}.
+ *
+ * <p>A point of interest is typically used by villagers to specify their workstation blocks, meeting zones and homes.
+ * Points of interest are also used by bees to specify where their bee hive is and nether portals to find existing portals.</p>
```

Comment:
Also, I'm suspecting that a second <p> mark might be necessary? Maybe not
```suggestion
 * <p>
 * A point of interest is typically used by villagers to specify their workstation blocks, meeting zones and homes.
 * Points of interest are also used by bees to specify where their bee hive is and nether portals to find existing portals.
```